Rubberduck.VBEEditor/ComManagement/TypeLibs/Abstract/ITypeInfoFunction.cs

```diff
@@ -21,6 +21,6 @@ public interface ITypeInfoFunction : IDisposable
         string Name { get; }
         int ParamCount { get; }
         PROCKIND ProcKind { get; }
-        void Dispose();
+        //void Dispose();
```

Comment:
Given this implements `IDisposable`, and as long the `Dispose()` is called, we can just delete it altogether.